Tests/AdyenTests/Adyen Tests/DropIn/DropInTests.swift

```diff
@@ -168,6 +168,25 @@ class DropInTests: XCTestCase {
         waitForExpectations(timeout: 15, handler: nil)
     }
 
+    func testOpenDropInWithNoOneClickPayment() {
+        let config = DropInComponent.Configuration(apiContext: Dummy.context, allowPreselectedPaymentView: false)
+
+        let paymentMethods = try! JSONDecoder().decode(PaymentMethods.self, from: DropInTests.paymentMethodsOneClick.data(using: .utf8)!)
+        sut = DropInComponent(paymentMethods: paymentMethods, configuration: config)
+
+        let root = UIViewController()
+        UIApplication.shared.keyWindow?.rootViewController = root
+        root.present(sut.viewController, animated: true, completion: nil)
+
+        let waitExpectation = expectation(description: "Expect DropIn to open")
+        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + .seconds(2)) {
```

Comment:
you can just use `wait(for: .seconds(2))`

Reply:
👍